src/index.html

```diff
@@ -22,6 +22,421 @@
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="https://snipboard.io/"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/e"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="https://snipboard.io/"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <!-- end-header -->
+
+    <!-- catalog -->
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__image"></div>
+        <div class="card__title">
+          <p class="card__title-p">
+            APPLE A1419 iMac 27" Retina
+            <br />
+            5K Monoblock (MNED2UA/A)
+          </p>
+        </div>
+        <div class="card__code">
+          <p class="card__code-p">Product code: 195434</p>
+        </div>
+
+        <div class="card__rate">
+          <div class="card-stars card-stars--4">
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+            <div class="card-stars__star"></div>
+          </div>
+          <div class="card__rate-review">
+            <p class="card__rate-review-p">Reviews: 5</p>
+          </div>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-numb">$2,199</p>
+        </div>
+
+        <div class="card__block">
+          <button
+            class="card__block-button"
+            data-qa="card-hover"
```

Comment:
The 'data-qa' attribute here is 'card-hover', which is inconsistent with other similar buttons where it is just 'hover'. Ensure consistency in attribute naming to avoid confusion.